pyannotate_tools/annotations/parse.py

```diff
@@ -188,7 +188,7 @@ def tokenize(s):
             tokens.append(Separator('->'))
             s = s[2:]
         else:
-            m = re.match(r'[-\w]+( *\. *[-/\w]*)*', s)
+            m = re.match(r'[-\w]+( *(\.|:) *[-/\w]*)*', s)
```

Comment:
Nit: Could you replace the spaces in the pattern with `\s` while you're at it? That makes it more readable IMO.